src/components/News.jsx

```diff
@@ -1,32 +1,63 @@
 import React from 'react';
 import styled from 'styled-components';
 import { graphql, StaticQuery } from 'gatsby';
+import Pager from 'react-pager';
 
 class News extends React.Component {
-  renderNewsItems = allItems => (
+  constructor(props) {
+    super(props);
+    this.pageAmount = 5; //Change this to change the amount of items per page
```

Comment:
I know it's not necessary to give the parent component control over this, but it just seems like something it should have control over, because it could vary between uses (if I understand what it does correctly).

Reply:
True that though